pkgs/by-name/bi/binwalk/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  fontconfig,
+  bzip2,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "binwalk";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ReFirmLabs";
+    repo = "binwalk";
+    rev = "4c71ce48bda3c8f0214d56ad6c18c3a833a8dd29";
+    hash = "sha256-5RgjspfTiiLyIyxr9BEHSlmjlWP02MKuSkMlQS482/E=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    fontconfig.dev
+    bzip2.out
```

```suggestion
    fontconfig
    bzip2
```
Isn't this choosen automatically?